src/controllers/user/display.js

```diff
@@ -13,7 +13,8 @@ define(["displaySettings", "userSettingsBuilder", "userSettings"], function(Disp
                 element: view.querySelector(".settingsContainer"),
                 userSettings: userSettings,
                 enableSaveButton: !1,
-                enableSaveConfirmation: !1
+                enableSaveConfirmation: !1,
+                autoFocus: true
```

Comment:
Where is this value checked?

Reply:
https://github.com/jellyfin/jellyfin-web/blob/c10f3feb3149d761c993a0c3b007c09c8a848f14/src/components/displaysettings/displaysettings.js#L295-L297